Nightly failure in phpstan

Build link: https://drone.owncloud.com/owncloud/activity/3357/3/4

Failure:

```bash
 ------ ----------------------------------------------------------------------------- 
  Line   lib/AppInfo/Application.php                                                  
 ------ ----------------------------------------------------------------------------- 
         Ignored error pattern #Method                                                
         Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface::dispatch\(\)  
         invoked with 2 parameters, 1 required.# in path                              
         /var/www/owncloud/server/apps/activity/lib/AppInfo/Application.php           
         was not matched in reported errors.                                          
 ------ ----------------------------------------------------------------------------- 


 [ERROR] Found 1 error                                                          

make: *** [Makefile:148: test-php-phpstan] Error 1
```

Related core issue: https://github.com/owncloud/core/issues/39630

--- a/phpstan.neon
+++ b/phpstan.neon
@@ -7,10 +7,6 @@ parameters:
       path: appinfo/routes.php
       count: 1
     -
-      message: '#Method Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface::dispatch\(\) invoked with 2 parameters, 1 required.#'
-      path: lib/AppInfo/Application.php
-      count: 1
-    -
       message: '#Comparison operation ">" between int<1, max> and 0 is always true.#'
       path: lib/Command/SendEmails.php
       count: 1

